demos/interactive_face_detection_demo/main.cpp

```diff
@@ -59,340 +47,524 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
+
+        // ------------------------------ Parsing and validating of input arguments --------------------------
+        if (!ParseAndCheckCommandLine(argc, argv)) {
+            return 0;
         }
-        // ---------------------------------------------------------------------------------------------------
-
-        // --------------------------- 2. Reading IR models and loading them to plugins ----------------------
-        // Disable dynamic batching for face detector as it processes one image at a time
-        Load(faceDetector).into(ie, FLAGS_d, false);
-        Load(ageGenderDetector).into(ie, FLAGS_d_ag, FLAGS_dyn_ag);
-        Load(headPoseDetector).into(ie, FLAGS_d_hp, FLAGS_dyn_hp);
-        Load(emotionsDetector).into(ie, FLAGS_d_em, FLAGS_dyn_em);
-        Load(facialLandmarksDetector).into(ie, FLAGS_d_lm, FLAGS_dyn_lm);
-        // ----------------------------------------------------------------------------------------------------
-
-        // --------------------------- 3. Doing inference -----------------------------------------------------
-        // Starting inference & calculating performance
-        slog::info << "Start inference " << slog::endl;
 
-        bool isFaceAnalyticsEnabled = ageGenderDetector.enabled() || headPoseDetector.enabled() ||
-                                      emotionsDetector.enabled() || facialLandmarksDetector.enabled();
+        if (FLAGS_n_ag != 0)
+            std::cout << "[ WARNING ] Option \"-num_batch_ag\" is not supported in this version of the demo.\n";
+        if (FLAGS_n_hp != 0)
+            std::cout << "[ WARNING ] Option \"-num_batch_hp\" is not supported in this version of the demo.\n";
+        if (FLAGS_n_em != 0)
+            std::cout << "[ WARNING ] Option \"-num_batch_em\" is not supported in this version of the demo.\n";
+        if (FLAGS_n_lm != 0)
+            std::cout << "[ WARNING ] Option \"-num_batch_lm\" is not supported in this version of the demo.\n";
+        if (FLAGS_dyn_ag != false)
+            std::cout << "[ WARNING ] Option \"-dyn_batch_ag\" is not supported in this version of the demo.\n";
+        if (FLAGS_dyn_hp != false)
+            std::cout << "[ WARNING ] Option \"-dyn_batch_hp\" is not supported in this version of the demo.\n";
+        if (FLAGS_dyn_em != false)
+            std::cout << "[ WARNING ] Option \"-dyn_batch_em\" is not supported in this version of the demo.\n";
+        if (FLAGS_dyn_lm != false)
+            std::cout << "[ WARNING ] Option \"-dyn_batch_lm\" is not supported in this version of the demo.\n";
+        if (FLAGS_pc != false)
+            std::cout << "[ WARNING ] Option \"-pc\" is not supported in this version of the demo.\n";
+        if (!FLAGS_c.empty())
+            std::cout << "[ WARNING ] Option \"-c\" is not supported in this version of the demo.\n";
+        if (!FLAGS_l.empty())
+            std::cout << "[ WARNING ] Option \"-l\" is not supported in this version of the demo.\n";
```

Comment:
`std::endl` is missing. If anything crashes, your output won't be flushed to the console.

Reply:
Done